base/steer/FairRun.cxx

```diff
@@ -79,14 +79,26 @@ void FairRun::AddTask(FairTask* t)
 {
   fTask->Add(t);
   fNTasks++;
+  StoreTaskNames(t);
+}
+//_____________________________________________________________________________
+void FairRun::StoreTaskNames(FairTask* t)
+{
+  if(t==NULL) return;
```

Comment:
Prefer `if(t == nullptr)` or just `if(!t)`.

Reply:
NULL is being used elsewhere in the same file. I'd leave it at that.